src/components/Medicine/MedicationAdministration/MedicineAdminForm.tsx

```diff
@@ -373,7 +373,7 @@ export const MedicineAdminForm: React.FC<MedicineAdminFormProps> = ({
           </Popover>
           <Input
             type="time"
-            className="w-[150px]"
+            className="w-full text-sm"
```

Comment:
is this needed? 🤔 let's NOT deviate from design system please!
```suggestion
            className="w-full"
```